assignments/callbacks.js

```diff
@@ -31,4 +37,32 @@ function removeDuplicates(array, cb) {
   // removeDuplicates removes all duplicate values from the given array.
   // Pass the duplicate free array to the callback function.
   // Do not mutate the original array.
+     return cb(array);
```

Comment:
you should remove duplicates inside of remove duplicates, and then pass the duplicate-free array to the cb